Dockerfile

```diff
@@ -1,25 +1,20 @@
-FROM harbor.k8s.libraries.psu.edu/library/ruby-3.1.2-node-16:20240701 as base
+FROM harbor.k8s.libraries.psu.edu/library/ruby-3.1.2-node-16:20240701 AS base
 # Isilon has issues with uid 2000 for some reason
 # change the app to run as 201
 ARG UID=201
-SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
+USER root
 WORKDIR /app
-
-RUN useradd -l -u $UID app -d /app
+RUN groupadd -g $UID app
+RUN useradd -l -u $UID app -g $UID -d /app
```

Comment:
it should be -g $GID

Reply:
I didn't define a GID variable so I just had it piggyback off of the GID value.  Would you add it ?

Reply:
Dropped this in for you... https://github.com/psu-libraries/researcher-metadata/pull/1019/commits/0cf0288ff308d28e74d7d4ee100333378782a1d2